### ISSUE TITLE
[Survey][React] onAfterRenderPage event works incorrectly

### ISSUE THREAD

**Issue by stsiushkevich** [surveyjs/surveyjs#1504]
* browser: Chrome
* browser version: Version 71.0.3578.98 (Official Build) (64-bit)
* surveyjs platform: React
"survey-react": "^1.0.63",
 "surveyjs-editor": "^1.0.63",
 "surveyjs-widgets": "^1.0.63"

### Are you requesting a feature, reporting a bug or asking a question?

A bug

### What is the current behavior?

Methods: `onCurrentPageChanged ` and `onAfterRenderPage `fire only once. They don't fire after I change `currentPageNo` prop.

### What is the expected behavior?

They should fire each time when I change `currentPageNo` prop. In the native JS version, there is no such issue.

#### Provide the test code and the tested page URL (if applicable)

``` javascript
<Survey.Survey
	ref={this.surveyRef}
	currentPageNo={step - 1}
	css={{
		pageTitle: "sv_p_title AssessmentForm-PageTitle",
		question: {
			mainRoot: "sv_q sv_qstn AssessmentForm-Question",
			title: "sv_q_title AssessmentForm-QuestionTitle"
		},
		panel: {
			title: "sv_p_title AssessmentForm-PanelTitle"
		},
		row: "sv_row AssessmentForm-Row"
	}}
	model={new Survey.Model(survey.value)}
	onAfterRenderPage={(survey, options) => {
		console.log(survey, options)
	}}
	onCurrentPageChanged={(survey, options) => {
		console.log(survey, options)
	}}
	onAfterRenderSurvey={(survey, options) => {
		survey.onCurrentPageChanged.add((survey, options) => {
			console.log(survey, options)
		})
	}}
	onAfterRenderPanel={this.onAfterRenderSurveyPanel}
/>
```
[Survey (1).txt](https://github.com/surveyjs/surveyjs/files/2748886/Survey.1.txt)

**Comment by dmitrykurmanov** [comment]
@stsiushkevich thank you for the reporting. Here an example: https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview please check it.

`onCurrentPageChanged` event works as expected, but `onAfterRenderPage` **fires only once**. 

So I think that the problem only in `onAfterRenderPage` event. In jQuery and Vue it works fine.

**Comment by dmitrykurmanov** [comment]
related to https://github.com/surveyjs/surveyjs/issues/664

**Comment by stsiushkevich** [comment]
> @stsiushkevich thank you for the reporting. Here an example: https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview please check it.
> 
> `onCurrentPageChanged` event works as expected, but `onAfterRenderPage` **fires only once**.
> 
> So I think that the problem only in `onAfterRenderPage` event. In jQuery and Vue it works fine.

When we use Survey's navigator - it works fine. But if we use `currentPageNo={x}` prop - `onCurrentPageChanged` method doesn't fire.

**Comment by dmitrykurmanov** [comment]
@stsiushkevich it is strange. I've opened console in the example above and typed `survey.currentPageNo = 2` and "onCurrentPageChanged" event fired

**Comment by stsiushkevich** [comment]
> @stsiushkevich I've fixed the problem with the `onAfterRenderPage` event and updated the example (included latest build):
> 
> https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview
> 
> what about `onCurrentPageChanged` event? Could you please modify the plunk to reproduce the issue?

There is another issue. When I pass the same surveyModel object `onAfterRenderPanel` fires many times for the same panel panel

**Comment by stsiushkevich** [comment]
> @stsiushkevich I've fixed the problem with the `onAfterRenderPage` event and updated the example (included latest build):
> 
> https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview
> 
> what about `onCurrentPageChanged` event? Could you please modify the plunk to reproduce the issue?

I found the issue source: `widgets.prettycheckbox(Survey, $);`

After this `onCurrentPageChanged` method (maybe and another too) wors incorrectly

**Comment by stsiushkevich** [comment]
> > @stsiushkevich I've fixed the problem with the `onAfterRenderPage` event and updated the example (included latest build):
> > https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview
> > what about `onCurrentPageChanged` event? Could you please modify the plunk to reproduce the issue?
> 
> There is another issue. When I pass the same surveyModel object `onAfterRenderPanel` **fires many times for the same panel panel**

I have prepared an example: https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview

Switch tabs and watch a count of `onAfterRenderPanel` calls

**Comment by stsiushkevich** [comment]
> > > @stsiushkevich I've fixed the problem with the `onAfterRenderPage` event and updated the example (included latest build):
> > > https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview
> > > what about `onCurrentPageChanged` event? Could you please modify the plunk to reproduce the issue?
> > 
> > 
> > There is another issue. When I pass the same surveyModel object `onAfterRenderPanel` **fires many times for the same panel panel**
> 
> I have prepared an example: https://plnkr.co/edit/kXddaSc9zD9BwCX7Z71w?p=preview
> 
> Please, switch tabs and watch a count of `onAfterRenderPanel` calls

**Comment by stsiushkevich** [comment]
> @stsiushkevich I've fixed the problem with the `onAfterRenderPage` event and updated the example (included latest build):
> 
> https://plnkr.co/edit/0ha2kRqhTWZq1UD95MVY?p=preview
> 
> what about `onCurrentPageChanged` event? Could you please modify the plunk to reproduce the issue?

Hello! Are you fixing the issue?

**Comment by dmitrykurmanov** [comment]
We have several issues here. I am going to create new issues.
